rpcd/playbooks/roles/rpc_maas/tasks/raxmon_agent_install.yml

```diff
@@ -38,6 +38,11 @@
     group: "root"
     mode: "0600"
 
+- name: Verify raxmon-entities-list does not fail
```

Comment:
Would be good to indicate that this task is for testing the MaaS credentials / ability to authenticate against the MaaS API.